udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -20,13 +20,55 @@ async function deleteCommentData() {
   await getCommentData;
   window.location.reload();
 }
+
 /** Creates an <li> element containing text. */
 function createCommentListItem(inputText) {
   const listElement = document.createElement('li');
   listElement.innerText = inputText;
   return listElement;
 }
 
+/** Creates a map with marker at UC San Diego */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 32.879838, lng: -117.232351}, zoom: 16});
+
+  const trexMarker = new google.maps.Marker({
```

Comment:
Out of curiosity, what does trex mean here?

Reply:
Changed to schoolMarker, original name was temporary.